go/vt/vtctl/workflow/server.go

```diff
@@ -3130,12 +3143,16 @@ func (s *Server) WorkflowSwitchTraffic(ctx context.Context, req *vtctldatapb.Wor
 			return nil, vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "cannot reverse traffic for multi-tenant migrations")
 		}
 	}
-	reason, err := s.canSwitch(ctx, ts, startState, direction, int64(maxReplicationLagAllowed.Seconds()), req.Shards)
+
+	ts.force = req.GetForce()
+
+	reason, err := s.canSwitch(ctx, ts, startState, direction, int64(maxReplicationLagAllowed.Seconds()), req.GetShards(), req.GetForce())
```

Comment:
It seems unnecessary to add an extra parameter to this func (which is called only from this one place), when `ts.force` is being set, and can be checked.

Reply:
Good catch! It’s a hold over from an earlier implementation. I’ll remove it.

Reply:
Done: https://github.com/vitessio/vitess/pull/16709/commits/28347604a5a218438b0b55bc499a2b2e286d55ef